src/main/java/org/jetbrains/bsp/bazel/server/BazelBspServer.java

```diff
@@ -567,7 +571,7 @@ private String getSourcesRoot(String uri) {
 
   private Either<ResponseError, CompileResult> buildTargetsWithBep(
       List<BuildTargetIdentifier> targets, List<String> extraFlags) {
-    List<String> args = Lists.newArrayList("build");
+    List<String> args = Lists.newArrayList(Constants.BAZEL_BUILD_COMMAND);
```

Comment:
Perhaps substitute with `ImmutableList.of()`?

Reply:
I agree, but there are another 15 random newArrayList and new ArrayList<>() in BazelBspServer so probably for another PR.

Reply:
Sure thing.